go.mod

```diff
@@ -19,15 +18,16 @@ require (
 	github.com/docker/docker-credential-helpers v0.6.3 // indirect
 	github.com/docker/go-connections v0.4.0 // indirect
 	github.com/morikuni/aec v1.0.0 // indirect
-	github.com/opencontainers/artifacts v0.0.0-20210209205009-a282023000bd
+	github.com/opencontainers/artifacts v0.0.0-20210209205009-a282023000bd // indirect
```

Comment:
We can remove this line as `github.com/opencontainers/artifacts` is used nowhere.